src/lib/pages/backoffice/Patron/PatronDetails/PatronDocumentRequests/__snapshots__/PatronDocumentRequests.test.js.snap

```diff
@@ -550,7 +550,7 @@ exports[`PatronDocumentRequests tests should render patron document requests 1`]
                                   data-label="Created on "
                                   data-test="4-documentRequest1"
                                 >
-                                  2018-01-01
+                                  1/1/2018
```

Comment:
I'm afraid that this will be updated depending on our locale settings, each time one of us with different locale runs the test

Reply:
Very good point... we can just "force" the locale in the tests, need to have a look!